src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public String toString() {
+        return color + " " + number;
```

Comment:
```suggestion
        return "Ball with color - " + color + " and number - " + number;
```
make toString method more informative